eclass/nodejs-mod.eclass

```diff
@@ -0,0 +1,110 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: nodejs-mod.eclass
+# @MAINTAINER:
+# Fco. Javier Félix <web@inode64.com>
+# @AUTHOR:
+# Fco. Javier Félix <web@inode64.com>
+# @SUPPORTED_EAPIS: 7 8
+# @BLURB: An eclass for build NodeJS projects
+# @DESCRIPTION:
+# An eclass providing functions to build NodeJS projects
+#
+# Changelog:
```

Comment:
Changelog? I would say credits or similar. :P

Reply:
this may be better.

work and ideas from:

Reply:
> this may be better.
> 
> work and ideas from:

Im fine with that